docs/resources/cbh_instance.md

```diff
@@ -62,6 +62,11 @@ The following arguments are supported:
 
   Changing this parameter will create a new resource.
 
+* `product_info` - (Required, List, ForceNew)
```

Comment:
How about add a description to this field

Reply:
done